app/assets/stylesheets/preact/sidebar-widget.scss

```diff
@@ -73,5 +73,23 @@
         }
       }
     }
+    .write {
+      display: block;
+      text-align: center;
+      padding: 8px 16px;
+      border-radius: 3px;
+      margin: 8px 0;
+      @include themeable(background, theme-top-bar-write-background, #1ab3a6);
```

Comment:
Is there a better idea to specify a new background color instead of setting it explicitly?

Reply:
Will it be changing during the campaign of She Coded, or do you mean in general for campaigns? Anyways, if you wanted to be able to change it without setting it explicitly, you'd have to set a CSS property on the style tag that has the `write` CSS class. But, this wouldn't handle the case of it being themeable.

Reply:
We discussed this as a team and decided to keep this css as is because we have too many unknowns about what we'll need for the future campaigns.